core/controllers/topic_viewer.py

```diff
@@ -31,6 +31,21 @@
 class TopicViewerPage(base.BaseHandler):
     """Renders the topic viewer page."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'classroom_url_fragment': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'topic_url_fragment': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
```

Comment:
Again, are there more rules for these?

Reply:
Hey @vojtechjelinek 
I have updated the rules. Kindly check them.